UI/multitrack-video-output.cpp

```diff
@@ -398,9 +398,9 @@ void MultitrackVideoOutput::PrepareStreaming(
 	auto multitrack_video_name =
 		QTStr("Basic.Settings.Stream.MultitrackVideoLabel");
 	if (obs_data_has_user_value(service_settings,
-				    "ertmp_multitrack_video_name")) {
```

Comment:
I feel like I fixed this at least 10 times or so 😅 juggling between twitch beta branches and PRs

thanks for fixing this 👍

Reply:
No problem.